### ISSUE TITLE
[BUG] Unlabeled buttons in app

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5740]
**Describe the bug**
This will be a bug to track Unlabeled buttons throughout app. 
1. [WCAG 2.4.6 Headings and Labels (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels.html#:~:text=This%20success%20criterion%20requires%20that,while%20failing%20Success%20Criterion%201.3.) - This criterion states that labels must be descriptive. If a page has three buttons labeled "Edit," a user (particularly a screen reader user) cannot distinguish them, making the labels not truly descriptive of their purpose
2. [ WCAG 4.1.2 Name, Role, Value (Level A) ](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)- This requires that all user interface components (including buttons) have a "programmatically determinable" name.

On Notifications screen, the Delete button on each card needs to have a unique label. _Since this repeats, I recommend something like using the title of the card in the aria label for the button for a unique one._ 

**To Reproduce**
Per email from Mark Hennessy, we are not needing steps to reproduce for every unlabeled button. 

**Expected behavior**
Every button needs a unique label for assistive technology users to discern their purpose/destination/function. 

**Screenshots**
The Bold Italic and Underline buttons are not labeled on Post and Post replies:

![Image](https://github.com/user-attachments/assets/f4139b8b-cb0b-4acc-96f6-4254dad44d5b)

**Smartphone (please complete the following information):**
 - Device: Google Pixel 8a
 - OS:  Android 16
 - Version 7.3.50

 - Device: iPhone 14+
 - OS:  18.6.2
 - Version 7.3.48

**Additional context**
I will be adding any other unlabeled buttons to this ticket as discussed prior. 

Additional example:

**Comment by acovher2** [comment]
<img width="426" height="926" alt="Image" src="https://github.com/user-attachments/assets/97e9ab59-202a-4fbb-8bf5-774861335d55" />
@TodorBachvarov This is another instance of unlabeled buttons. 
The aria-labels of this can be common sense labels like (in order from left to right): 'rotate image left' 'rotate image right' 'cancel upload' 'complete upload'
@vburgett  Are you okay with those labels for these buttons? 
Additionally, technically for the image upload flow this is a Showstopper since they cannot complete the upload without knowing what button to select. I'm not sure if the whole ticket should be upgraded.

**Comment by TodorBachvarov** [comment]
@acovher2, 
- The NotificationCard delete button is fixed. Fix provided in version 7.3.52/8.0.5
- Settings Panel: I'm not sure what is the problem in this panel. All buttons seems to have labels. 
- Image Selection/Edit Panel: This is external library that we use so it is not so trivial to update the Labels of the Rotate/Cancel/Complete buttons. I can suggest this issue to be described as separate Git Issue, so we can track it and provide additional more complex fix (for example fork the library and apply custom labeling), and mark it as showstopper if needed. Lets keep this ticket open, or close if it's done for now and reopen in future if you find other instances of this defect.

**Comment by acovher2** [comment]
@TodorBachvarov 

1. The NotificationCard delete button is fixed. Fix provided in version 7.3.52/8.0.5 - **Will retest when that version is available. What is the expected date? **
2. Settings Panel: I'm not sure what is the problem in this panel. All buttons seem to have labels. - **These were listed in the _Device_ section and put there to show what version was being tested. There is no issue. **
3. Image Selection/Edit Panel: This is external library that we use so it is not so trivial to update the Labels of the Rotate/Cancel/Complete buttons. I can suggest this issue to be described as separate Git Issue, so we can track it and provide additional more complex fix (for example fork the library and apply custom labeling), and mark it as showstopper if needed. Lets keep this ticket open, or close if it's done for now and reopen in future if you find other instances of this defect. - **If I am understanding correctly, because this uses some external library you can not update this as simply as the "Delete" icons?  If so Please provide more information as to what library is preventing this fix so that I may include it in the separate Github? Thank you. **

**Comment by TodorBachvarov** [comment]
@acovher2, yes this is correct. The external library that we use is named image_cropping (image_cropping: ^0.0.16).
